package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "embeddings-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arangoml/embeddings-service.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
You probably want to remove the license field. You can actually omit most of the unused fields. Just add `"private": true` so npm knows this isn't meant to be published and complains a little less. Foxx ignores the package.json's contents entirely.

Reply:
Fixed!